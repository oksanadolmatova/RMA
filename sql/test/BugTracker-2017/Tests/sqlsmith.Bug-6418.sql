CREATE TABLE PARTSUPP_6418 ( PS_PARTKEY     INTEGER NOT NULL,
                             PS_SUPPKEY     INTEGER NOT NULL,
                             PS_AVAILQTY    INTEGER NOT NULL,
                             PS_SUPPLYCOST  DECIMAL(15,2)  NOT NULL,
                             PS_COMMENT     VARCHAR(199) NOT NULL );
select  
  ref_2.key_type_id as c0, 
  ref_3.ps_availqty as c1, 
  sample_0.name as c2
from 
  tmp.idxs as sample_0
      right join sys.sequences as sample_1
      on ((true) 
          or ((sample_0.name is NULL) 
            or (sample_0.type is not NULL)))
    inner join sys.key_types as ref_2
      left join sys.partsupp_6418 as ref_3
      on (ref_2.key_type_id is not NULL)
    on (sample_1.schema_id is NULL)
where true
limit 116;

drop table partsupp_6418;
