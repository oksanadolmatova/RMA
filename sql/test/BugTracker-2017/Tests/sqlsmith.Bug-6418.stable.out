stdout of test 'sqlsmith.Bug-6418` in directory 'sql/test/BugTracker-2017` itself:


# 23:28:47 >  
# 23:28:47 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=30700" "--set" "mapi_usock=/var/tmp/mtest-25365/.s.monetdb.30700" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 23:28:47 >  

# MonetDB 5 server v11.27.6
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 7.330 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:30700/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-25365/.s.monetdb.30700
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 23:28:48 >  
# 23:28:48 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-25365" "--port=30700"
# 23:28:48 >  

#CREATE TABLE PARTSUPP_6418 ( PS_PARTKEY     INTEGER NOT NULL,
#                             PS_SUPPKEY     INTEGER NOT NULL,
#                             PS_AVAILQTY    INTEGER NOT NULL,
#                             PS_SUPPLYCOST  DECIMAL(15,2)  NOT NULL,
#                             PS_COMMENT     VARCHAR(199) NOT NULL );
#select  
#  ref_2.key_type_id as c0, 
#  ref_3.ps_availqty as c1, 
#  sample_0.name as c2
#from 
#  tmp.idxs as sample_0
#      right join sys.sequences as sample_1
#      on ((true) 
#          or ((sample_0.name is NULL) 
#            or (sample_0.type is not NULL)))
#    inner join sys.key_types as ref_2
#      left join sys.partsupp_6418 as ref_3
#      on (ref_2.key_type_id is not NULL)
#    on (sample_1.schema_id is NULL)
#where true
#limit 116;
% .L2,	.L4,	.L6 # table_name
% c0,	c1,	c2 # name
% smallint,	int,	varchar # type
% 1,	1,	0 # length
#drop table partsupp_6418;

# 23:28:48 >  
# 23:28:48 >  "Done."
# 23:28:48 >  

